label_studio/users/serializers.py

```diff
@@ -18,7 +19,26 @@ def get_avatar(self, user):
         return user.avatar_url
 
     def get_initials(self, user):
-        return user.get_initials()
+        if flag_set('fflag_feat_all_optic_114_soft_delete_for_churned_employees', user=user):
+            return user.get_initials(self._is_deleted(user))
+        else:
+            return user.get_initials()
+
+    def _is_deleted(self, instance):
```

Comment:
could `instance` be `user` or are there cases where this is called and it isn't a user object?

Reply:
Its my understanding that `get_` methods in a DRF serializer receive an instance of the the model declared in their Meta class.  Here's an example from DRF docs:
```
class UserSerializer(serializers.ModelSerializer):
    days_since_joined = serializers.SerializerMethodField()

    class Meta:
        model = User
        fields = '__all__'

    def get_days_since_joined(self, obj):
        return (now() - obj.date_joined).days
```

I'll rename all those params so they're consistent though.